### PR TITLE
CoL Map Update: Fixing Old/Adding New Backstreets Layouts (also 2 new rooms)

### PR DESCRIPTION
**Pull request by EnderMage99**
## About The Pull Request

This PR adds 2 new backstreets rooms, fixes the Old L-Corp map, (The one that I have made before.), and adds new things to the L-Corp map (The one that has a big hole in the middle) and the standard map. Also, This PR fixes other maps who could not load backstreets rooms due to them not having enough room.

## Why It's Good For The Game

First, It is new content for players to explore. Second, Bug fixes are good.

## Changelog
:cl:
add: 2 new backstreets room, new backstreets layout, and new objects on the older l-corp map and standard map
fix: fixed the problem where some backstreets rooms didn't have enough space to load in, and the newer l-corp map
/:cl:

### CODE REVIEW

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 1]
```diff
@@ -0,0 +1,2209 @@
...
+/obj/item/slimepotion/slime/docility,
+/obj/item/slimepotion/slime/docility,
+/obj/item/slimepotion/slime/docility,
+/turf/open/floor/plasteel/white,
+/area/city/backstreets_room)
+"ci" = (
+/obj/structure/chair/office,
+/obj/machinery/light{
+	dir = 1;
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:75]: Me when this isn't a string (Mapdiffbot2's output is helpfull)
```suggestion
	color = "#32a852"
```

**Comment by Gboster-0** [comment]
What an interesting name for a new backstreets layout, wonder how that came about

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm (hunk 1 above)
Comment (EnderMage99) [reply _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:75]: Thanks, I just noticed that I missed a few of those and tried to fix them in the Fixing Slime Commit. But looks like I still missed a few.

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 2]
```diff
@@ -0,0 +1,2209 @@
...
+/obj/structure/table/reinforced,
+/turf/open/floor/plasteel/shuttle{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"hK" = (
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/machinery/light{
+	dir = 1;
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:269]: here too
```suggestion
	color = "#32a852"
```

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 3]
```diff
@@ -0,0 +1,2209 @@
...
+	},
+/obj/effect/landmark/cratespawn/corpo,
+/obj/effect/decal/cleanable/cobweb,
+/turf/open/floor/facility/dark,
+/area/city/backstreets_room)
+"il" = (
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/machinery/light{
+	dir = 1;
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:279]: and here
```suggestion
	color = "#32a852"
```

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 4]
```diff
@@ -0,0 +1,2209 @@
...
+/obj/item/slimecross/industrial/grey,
+/obj/item/slimecross/industrial/grey,
+/mob/living/simple_animal/hostile/ordeal/steel_dawn,
+/turf/open/floor/plasteel/shuttle{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"kD" = (
+/mob/living/simple_animal/slime/pet{
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:352]:
```suggestion
	color = "#32a852"
```

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 5]
```diff
@@ -0,0 +1,2209 @@
...
+/area/city/backstreets_room)
+"nc" = (
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/effect/gibspawner/human/bodypartless,
+/obj/item/trash/semki,
+/turf/open/floor/facility/dark,
+/area/city/backstreets_room)
+"nd" = (
+/mob/living/simple_animal/slime/pet{
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:482]:
```suggestion
	color = "#32a852"
```

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 6]
```diff
@@ -0,0 +1,2209 @@
...
+	},
+/obj/item/trash/energybar,
+/turf/open/floor/plating{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"ol" = (
+/obj/machinery/light{
+	dir = 1;
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:512]:
```suggestion
	color = "#32a852"
```

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 7]
```diff
@@ -0,0 +1,2209 @@
...
+/turf/open/floor/plasteel/shuttle{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"qj" = (
+/obj/machinery/computer/camera_advanced/xenobio{
+	dir = 1
+	},
+/obj/machinery/light{
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:594]:
```suggestion
	color = "#32a852"
```

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 8]
```diff
@@ -0,0 +1,2209 @@
...
+/obj/effect/landmark/cratespawn/corpo,
+/obj/machinery/light/broken{
+	dir = 1
+	},
+/turf/open/floor/facility/dark,
+/area/city/backstreets_room)
+"sH" = (
+/obj/effect/turf_decal/siding/purple,
+/mob/living/simple_animal/slime/pet{
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:646]:
```suggestion
	color = "#32a852"
```

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 9]
```diff
@@ -0,0 +1,2209 @@
...
+/turf/open/floor/plasteel/shuttle{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"sY" = (
+/obj/structure/plaque/static_plaque{
+	pixel_x = 32
+	},
+/mob/living/simple_animal/slime/pet{
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:657]:
```suggestion
	color = "#32a852"
```

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 10]
```diff
@@ -0,0 +1,2209 @@
...
+/obj/effect/turf_decal/siding/purple{
+	dir = 1
+	},
+/turf/open/floor/plating{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"xQ" = (
+/obj/machinery/light{
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:825]:
```suggestion
	color = "#32a852"
```

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 11]
```diff
@@ -0,0 +1,2209 @@
...
+/obj/effect/turf_decal/siding/purple/corner{
+	dir = 1
+	},
+/turf/open/floor/plasteel/shuttle{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"BJ" = (
+/obj/machinery/light{
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:989]:
```suggestion
	color = "#32a852"
```

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 12]
```diff
@@ -0,0 +1,2209 @@
...
+/obj/item/reagent_containers/glass/beaker/bluespace,
+/obj/item/reagent_containers/glass/beaker/bluespace,
+/obj/item/reagent_containers/glass/beaker/bluespace,
+/turf/open/floor/plating{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"BW" = (
+/obj/machinery/light{
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:1009]:
```suggestion
	color = "#32a852"
```

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 13]
```diff
@@ -0,0 +1,2209 @@
...
+/obj/structure/spider/stickyweb/genetic,
+/mob/living/simple_animal/hostile/ordeal/steel_dawn,
+/turf/open/floor/plasteel/shuttle{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"Fw" = (
+/obj/structure/sign/departments/xenobio,
+/turf/closed/indestructible/reinforced{
+	color = #42f545
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:1153]:
```suggestion
	color = "#42f545"
```

File: _maps/RandomRooms/backstreets/bossroom/slime_office.dmm [hunk 14]
```diff
@@ -0,0 +1,2209 @@
...
+	color = "#006400";
+	dir = 9
+	},
+/turf/open/floor/plating{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"Ry" = (
+/mob/living/simple_animal/slime/pet{
+	color = #32a852
```
Comment (Gboster-0) [inline _maps/RandomRooms/backstreets/bossroom/slime_office.dmm:1514]:
```suggestion
	color = "#32a852"
```

**Review by Gboster-0 — COMMENTED** [review]
In hell we lament

File: ModularTegustation/tegu_items/backstreets/templates.dm [hunk 15]
```diff
@@ -28,7 +28,12 @@
...
 
-/datum/map_template/random_room/backstreet_template/lcorp_ruin
-	name = "L-Corp Ruin Template"
-	room_id = "lcorp_ruin"
+/datum/map_template/random_room/backstreet_template/abandoned_lcorp
+	name = "Abandoned L-Corp Template"
+	room_id = "abandoned_lcorp"
 	mappath = "_maps/RandomRooms/backstreetlayout/lcorp_backstreets.dmm"
+
+/datum/map_template/random_room/backstreet_template/la_mancha_land
```
Comment (Kitsunemitsu) [inline ModularTegustation/tegu_items/backstreets/templates.dm:36]: Would like to ask to change this one's weight from 10 to 5, that means that it shows up half as often.

It's spectacular and flashy, but a little too different to be on par with the others.

**Review by Kitsunemitsu — APPROVED** [review]
Approving this PR. Looks excellent to me.

**Pull request merged**